pkg/apic/servicebuilder_test.go

```diff
@@ -100,22 +100,31 @@ func TestServiceBodySetters(t *testing.T) {
 }
 
 func TestServiceBodyWithParseError(t *testing.T) {
+	t.Logf("12345dale")
 	serviceBuilder := NewServiceBodyBuilder()
 	_, err := serviceBuilder.SetResourceType(Oas3).SetAPISpec([]byte("{\"test\":\"123\"}")).Build()
+	t.Logf("2")
+	t.Log(err)
 	assert.NotNil(t, err)
 
-	_, err = serviceBuilder.SetResourceType(Oas2).SetAPISpec([]byte("{\"test\":\"123\"}")).Build()
-	assert.NotNil(t, err)
+	// t.Logf("3")
+	// _, err = serviceBuilder.SetResourceType(Oas2).SetAPISpec([]byte("{\"test\":\"123\"}")).Build()
+	// t.Log(err)
```

Comment:
All these logs and commented out code needed?

Reply:
No. Ignore this PR for now. I'm trying to figure out why a few sonar-preview tests don't work on the builders.